src/cloudant/_messages.py

```diff
@@ -16,17 +16,69 @@
+         'either \"json\" or \"text\".',
+    104: 'A single query/q parameter is required.  Found: {0}',
+    105: 'Invalid argument: {0}',
+    106: 'Argument {0} is not an instance of expected type: {1}',
+    # Design document
+    107: 'View {0} already exists in this design doc.',
+    108: 'An index with name {0} already exists in this design doc.',
+    109: 'A list with name {0} already exists in this design doc.',
+    110: 'A show function with name {0} already exists in this design doc.',
+    111: 'View {0} does not exist in this design doc.',
+    112: 'An index with name {0} does not exist in this design doc.',
+    113: 'A list with name {0} does not exist in this design doc.',
+    114: 'A show function with name {0} does not exist in this design doc.',
+    # Feed
+    115: 'Error converting argument {0}: {1}',
+    116: 'Invalid argument {0}',  # py_to_couch_validate
+    117: 'Argument {0} not instance of expected type: {1}',
+    118: 'Argument {0} must be > 0. Found: {1}',
+    119: 'Invalid value ({0}) for feed option.  Must be one of {1}',
+    120: 'Invalid value ({0}) for style option.  Must be main_only, '
+         'or all_docs.',
+    121: 'Invalid infinite feed option: {0}.  Must be set to continuous.',
+    # Index
+    122: 'The design document id: {0} is not a string.',
+    123: 'The index name: {0} is not a string.',
+    124: '{0} provided as argument(s).  A JSON index requires that '
+         'only a \'fields\' argument is provided.',
+    125: 'Deleting an index requires a design document id be provided.',
+    126: 'Deleting an index requires an index name be provided.',
+    127: 'Invalid argument: {0}',
+    128: 'Argument {0} is not an instance of expected type: {1}',
+    # Query
+    129: 'Invalid argument: {0}',
+    130: 'Argument {0} is not an instance of expected type: {1}',
+    131: 'No selector in the query or the selector was empty.  '
+         'Add a selector to define the query and retry.',
+    # View
+    132: 'The map property must be a dictionary.',
+    133: 'The reduce property must be a string.',
+    # Common_util
+    134: 'Key list element not of expected type: {0}',
+    135: 'Invalid value for stale option {0} must be ok or update_after.',
+    136: 'Error converting argument {0}: {1}'
+}
+
 CLIENT = {
     100: 'A general Cloudant client exception was raised.',
-    101: 'Value must be set to a Database object. Found type: {0}.',
+    101: 'Value must be set to a Database object. Found type: {0}',
     102: 'You must provide a url or an account.',
     404: 'Database {0} does not exist. Verify that the client is valid and try again.',
     409: 'Database {0} already exists.'
 }
 
 DATABASE = {
     100: 'A general Cloudant database exception was raised.',
-    101: 'Unexpected index type. Found: {0}.',
+    101: 'Unexpected index type. Found: {0}',
     400: 'Invalid database name during creation. Found: {0}',
     401: 'Unauthorized to create database {0}.',
```

Comment:
The period at the end of this one should be removed to conform with the style used for the rest.